### PR TITLE
[MRG] update link to centos installation script

### LINKED ISSUE #322: CentOS installation script not found
The instruction in [this page](https://jonescompneurolab.github.io/hnn/installer/centos/) uses an invalid URL for installation script.

### PR DESCRIPTION
**Pull request by rythorpe**
closes #322

### CODE REVIEW

**Comment by rythorpe** [comment]
I've already testing the links to make sure my changes download the desired installation bash scripts. I'm trusting that the scripts themselves still run correctly. Feel free to rebase and merge when you're ready @jasmainak @ntolley!

File: installer/centos/README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ The script below assumes that it can update OS packages for python and prerequis
 * CentOS 7: [centos7-installer.sh](centos7-installer.sh)
 
     ```bash
-    curl --remote-name https://raw.githubusercontent.com/jonescompneurolab/hnn/master/installer/ubuntu/hnn-centos-7.sh
```
Comment (ntolley) [inline installer/centos/README.md:8]: I hope this didn't take too long to spot :smile:

File: installer/centos/README.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ The script below assumes that it can update OS packages for python and prerequis
...
+    curl --remote-name https://raw.githubusercontent.com/jonescompneurolab/hnn/master/installer/centos/hnn-centos7.sh
     bash hnn-centos7.sh
     ```
 
 * CentOS 6 (no longer maintained): [centos6-installer.sh](centos6-installer.sh)
```
Comment (ntolley) [inline installer/centos/README.md:12]:
```suggestion
* CentOS 6 (no longer maintained): [hnn-centos6.sh](hnn-centos6.sh)
```
Fixes the hyperlinks

**Comment by ntolley** [comment]
Thanks @rythorpe ! Also some of the hyperlinks direct to a non-existing file. Happy to merge once those are updated.

File: installer/centos/README.md (hunk 1 above)
Comment (rythorpe) [reply installer/centos/README.md:8]: Longer than it should have... :cry:

**Comment by ntolley** [comment]
And it's in! Thanks @chenghuzi for spotting the broken link

**Pull request merged**